src/KeyPair.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Tighten\SolanaPhpSdk;
+
+use Tighten\SolanaPhpSdk\Exceptions\TodoException;
+use Tighten\SolanaPhpSdk\Util\Ed25519Keypair;
+
+/**
+ * An account keypair used for signing transactions.
+ */
+class KeyPair
+{
+    protected Ed25519Keypair $_keypair;
+
+    /**
+     * @param $secretKey
+     */
```

Comment:
Wrong docblock--why I love dropping docblocks when they're not needed :D

Reply:
😂 - let me sleep on them please. I do LOVE them for the visual. We can remove them later if they prove to be a pain in the butt to keep in sync